File: Homework/Unit_test/Python/Calculator.py
class Calculator:
    def sum(self, number1, number2):
        return number1 + number2
    

    def subtract(self, number1, number2):
        return number1 - number2
    
    def multiply(self, number1, number2):
        return number1 * number2
    
    def divide(self, number1, number2):
        if number2 == 0:
            raise ValueError('You cannot divide by zero')
        return number1 / number2

File: Homework/Unit_test/Js/Calculator.js
export class Calculator {
    sum(number1, number2) {
        return number1 + number2;
    }

    subtract(number1, number2) {
        return number1 - number2;
    }

    multiply(number1, number2) {
        return number1 * number2;
    }

    divide(number1, number2) {
        if (number2 === 0) {
            throw new Error("Cannot divide by zero");
        }
        return number1 / number2;
    }
}
